controllers/auth/index.js

```diff
@@ -6,37 +6,36 @@ const SignUp = async (req, res) => {
     let toAddUser = req.body;
 
     if (!toAddUser || !toAddUser.name || !toAddUser.email) {
-      return res.status(400).send({ message: "Not vaild Request Data" });
+      return res.status(400).send({ message: "insufficient data" });
     }
 
-    const UserByName = await UserModel.findOne({ name: toAddUser.name.trim() });
-    const UserByEmail = await UserModel.findOne({
+    const hasUser = await UserModel.findOne({
```

Comment:
we don't need to check if a user with the same email exits or not as MongoDB will handle it (we have a unique in over schema for email). I forget to remove it in my code.